composables/transaction/transactionBurn.ts

```diff
@@ -107,13 +107,42 @@ export function execBurnCollection(
     errorMessage,
   }: ExecuteTransactionParams) => void,
 ) {
+  const collectionId = params.collectionId.toString()
+
   try {
-    const cb = api.tx.nfts.destroy
+    if (params.urlPrefix === 'rmrk') {
+      executeTransaction({
+        cb: api.tx.system.remark,
+        arg: [createInteraction(Interaction.CONSUME, collectionId, '')],
+      })
+    }
 
-    executeTransaction({
-      cb,
-      arg: [params.collectionId.toString(), {}],
-    })
+    if (params.urlPrefix === 'ksm') {
+      executeTransaction({
+        cb: api.tx.system.remark,
+        arg: [collectionId, {}],
```

Comment:
is this how to destroy collection on ksm?

Reply:
yes, but idk if it's working I think I need kodadot/packages#180 first

ref https://github.com/rmrk-team/rmrk-substrate/blob/8c74166627c496435e2dff5dbea03a55d8fec764/pallets/rmrk-core/src/lib.rs#L574